src/hooks/useReadBalanceOf.ts

```diff
@@ -16,5 +16,6 @@ export function useReadBalanceOf(tokenAddress: EvmAddress) {
         { args: [accountEvm as EvmAddress] },
         tokenAddress.toString() as EvmAddress,
       ),
+    initialData: () => BigInt(0),
```

Comment:
I don't know if that's makes sense as we should always count on pending state in time balance still fetching, rather than show default one.

Reply:
My motivation here was that we use formatting balances functions, and default result of query until loaded is `undefined`, so I did not want expand those formatters to support the undefined too.

We still have isPending and isLoading in our queries, but we don't use them for now. In the future we can come up with some "show balance" component, which would both show the loading state and then format the values only when they arrive, so we can drop the initial data then.